digicampipe/scripts/spe.py

```diff
@@ -221,18 +225,18 @@ def find_pulse_3(events):
             axis=1,
             mode='constant',
         )
-        pulse_mask[:, 4:-4] = (
+        pulse_mask[:, 6:-6] = (
```

Comment:
why is this smaller now ?

What can you get rid of `[:, 5:-5]` ? or is this method causing issues at boundary ?

Reply:
Well ... when integrating with an integration window of size e.g. 7 .. then half of the window is outside the sampling area. So one usually does not even want to consider pulses too close to the border.